src/components/routing/guards/SessionGuard.tsx

```diff
@@ -24,7 +24,11 @@ export const SessionGuard = observer((props: SessionGuardProps) => {
       hasCurrentSession()
         .then((value) => {
           if (!value && location.pathname !== Paths.CREATE_PLAYER) {
-            navigate(Paths.CREATE_PLAYER)
+            navigate(
```

Comment:
Test what happens if you have no session and open /games/:gameId
Hopefully it does not crash if you don't have a participation.

Reply:
Get directed to createplayer, then white-screen with error: XML-Verarbeitungsfehler: Kein Wurzel-Element gefunden
Adresse: https://screw-your-neighbor-server.herokuapp.com/auth/session

Reply:
XML-Verarbeitungsfehler: Kein Wurzel-Element passiert auch sonst wenn du nicht eingeloggt bist.
Das sollte nicht der grund sein für den white screen

Reply:
@BacLuc Lütfen Almanca Kullanmayalım. Şu anda nasıl hissediyorsanız, ben de öyle hissediyorum. :) Lütfen işleri kolaylaştıralım ve resmi dil olan İngilizceyi tercih edelim. Rica ediyorum :)